src/lib.rs

```diff
@@ -28,6 +28,9 @@ pub async fn run(core_happ: &config::Happ, config: &config::Config) -> Result<()
     let hbs_connect = HbsClient::connect()?;
     let kyc_level = hbs_connect.get_kyc_level().await;
     debug!("Got kyc level {:?}", &kyc_level);
+    let jurisdiction = hbs_connect.get_jurisdiction().await;
```

Comment:
Let's combine these two calls so that you don't need to call hbs twice for the same call